lib/widgets/login.dart

```diff
@@ -161,30 +183,61 @@ class _EmailPasswordLoginPageState extends State<EmailPasswordLoginPage> {
   Widget build(BuildContext context) {
     assert(!PerAccountStoreWidget.debugExistsOf(context));
     return Scaffold(
-      appBar: AppBar(title: const Text('Log in')),
+      appBar: AppBar(title: const Text('Log in'),
+        bottom: _inProgress
+          ? const PreferredSize(preferredSize: Size.fromHeight(4),
+              child: LinearProgressIndicator(minHeight: 4))
```

Comment:
This clashes with the circular progress indicator that #87 puts on the previous screen.

I think either is fine; should just make them consistent.

This linear indicator is probably simpler from a perspective of where to put the indicator.

Reply:
Yeah, a `CircularProgressIndicator.adaptive` isn't good for putting on an `ElevatedButton.icon`. There was a layout issue on Android that didn't seem easy to solve. Let's go with the linear one.